Description: Add linker flags for static libtgvoip
 Only libtgvoip.a is now available in Debian, while the libtgvoip-dev package
 still supplies the tgvoip.pc file for shared variant.
Forwarded: not-needed
Author: Nicholas Guriev <guriev-ns@ya.ru>
Last-Update: Tue, 25 Aug 2020 23:14:30 +0300

--- a/Telegram/cmake/lib_tgvoip.cmake
+++ b/Telegram/cmake/lib_tgvoip.cmake
@@ -11,7 +11,7 @@ if (TDESKTOP_USE_PACKAGED_TGVOIP AND NOT
     find_package(PkgConfig REQUIRED)
     pkg_check_modules(TGVOIP REQUIRED IMPORTED_TARGET tgvoip)
 
-    target_link_libraries(lib_tgvoip INTERFACE PkgConfig::TGVOIP)
+    target_link_libraries(lib_tgvoip INTERFACE PkgConfig::TGVOIP ${TGVOIP_STATIC_LDFLAGS})
 else()
     add_library(lib_tgvoip STATIC)
     init_target(lib_tgvoip)
